packages/EasyCore/src/Doctrine/Subscribers/EntityEventSubscriberInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Doctrine\Subscribers;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\ORM\Event\OnFlushEventArgs;
+use Doctrine\ORM\Event\PostFlushEventArgs;
+
+interface EntityEventSubscriberInterface extends EventSubscriber
+{
+    public function getSubscribedEvents(): array;
```

Comment:
PHPDoc are required for iterable arguments/return types.

Reply:
Done